source/parameter_acceptor.cc

```diff
@@ -380,12 +381,17 @@ MYV(std::vector<unsigned int>, ";")
 namespace
 {
   template<class T>
-  inline bool to_boost_any(const std::string &entry, boost::any &boost_parameter)
+  inline bool to_boost_any(const std::string &entry,
+                           const Patterns::PatternBase &pattern,
+                           boost::any &boost_parameter)
   {
     if (boost_parameter.type() == typeid(T *))
       {
         T &parameter = *(boost::any_cast<T *>(boost_parameter));
         parameter = ParameterAcceptor::to_type<T>(entry);
+//        Assert(pattern.match(entry),
+//               ExcMessage("The entry '"+entry+"' does not match"
+//                          " the pattern you specified: "+pattern.description()));
```

Comment:
This is currently giving me problems. Should work in principle, but seems not to. :(